hw/dv/tools/dvsim/testplans/sec_cm_count_testplan.hjson

```diff
@@ -0,0 +1,22 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: hardened_counter_check
+      desc: ''' Verify design behavior is correct when the harden counter is changed to an
+            unexpected value.
+
+            Stimulus:
+            - At the falling edge (non-active edge), force the counter to a different value.
+            - Randomly flip the value back to any other value to ensure the error is latched and
+              won't go away until reset.
```

Comment:
```suggestion
            Stimulus:
            - Randomly pick one of the two counters. 
            - At the falling edge (non-active edge), force-deposit the counter to a different value than expected.
            - On the next cycle, the violation of hardened counter property should generate an alert.
            - Repeat for ALL prim_count instance in the DUT. 
```

Reply:
I actually don't randomly pick the counters. just always use one counter, which makes the sequence less complex. 
Also alert may not arrive right on the next cycle, as alert is async
Updated the stimulus.